x-pack/legacy/plugins/siem/public/components/page/hosts/first_last_seen_host/index.test.tsx

```diff
@@ -12,26 +12,25 @@ import { render, act } from '@testing-library/react';
 import { mockFirstLastSeenHostQuery } from '../../../../containers/hosts/first_last_seen/mock';
 import { wait } from '../../../../lib/helpers';
 import { TestProviders } from '../../../../mock';
-import '../../../../mock/ui_settings';
 
 import { FirstLastSeenHost, FirstLastSeenHostType } from '.';
 
-jest.mock('../../../../lib/settings/use_kibana_ui_setting');
-
-// Suppress warnings about "react-apollo" until we migrate to apollo@3
-/* eslint-disable no-console */
-const originalError = console.error;
-beforeAll(() => {
-  console.error = jest.fn();
-});
-afterAll(() => {
-  console.error = originalError;
-});
+jest.mock('../../../../lib/kibana');
 
 describe('FirstLastSeen Component', () => {
   const firstSeen = 'Apr 8, 2019 @ 16:09:40.692';
   const lastSeen = 'Apr 8, 2019 @ 18:35:45.064';
 
+  // Suppress warnings about "react-apollo" until we migrate to apollo@3
+  /* eslint-disable no-console */
```

Comment:
I think we are the only ones left with react-apollo at this point :-) 

I see some left overs in `Infra` but I would imagine we are going to take over ownership unless another plugin starts using it.